src/payment/checkout/payment-form/PaymentForm.jsx

```diff
@@ -217,40 +214,12 @@ export class PaymentFormComponent extends React.Component {
           <CardDetails
             disabled={disabled}
           />
-          <div className="row justify-content-end">
-            <div className="col-lg-6 form-group">
-              {
-                loading || isQuantityUpdating || !window.microform ? (
-                  <div className="skeleton btn btn-block btn-lg">&nbsp;</div>
-                ) : (
-                  <StatefulButton
-                    type="submit"
-                    id="placeOrderButton"
-                    variant="primary"
-                    size="lg"
-                    block
-                    state={submitButtonState}
-                    onClick={this.props.onSubmitButtonClick}
-                    labels={{
-                      default: (
-                        <FormattedMessage
-                          id="payment.form.submit.button.text"
-                          defaultMessage="Place Order"
-                          description="The label for the payment form submit button"
-                        />
-                      ),
-                    }}
-                    icons={{
-                      processing: (
-                        <span className="button-spinner-icon" />
-                      ),
-                    }}
-                    disabledStates={['processing', 'disabled']}
-                  />
-                )
-              }
-            </div>
-          </div>
+          <PlaceOrderButton
```

Comment:
It looks like you're replacing a Paragon stateful button with one provided by Stripe. Is that right? I note this because conveying state info to screen reader users is a bit tricky.  No specific requests but something I'll want to test more.

Reply:
@wittjeff apologies for the delay in this response! We just refactored the existing button to its own component, so it can be used by different parent components. The new component still uses Paragon's `StatefulButton`